crates/rspack_plugin_javascript/src/visitors/dependency/common_js_export_scanner.rs

```diff
@@ -36,6 +37,7 @@ pub struct CommonJsExportDependencyScanner<'a> {
   stmt_level: u32,
   last_stmt_is_expr_stmt: bool,
   is_top_level: bool,
+  ignored: &'a mut Vec<DependencyLocation>,
```

Comment:
I don't think that ﻿`ignored` is an optimal solution. Could we perhaps align the behavior with that of webpack?

Reply:
If we want to align with webpack, we need full implement of parser hooks. In webpack, JavascriptParser use SyncBailHook, means that if some plugin which tap these hooks and return anthing but not `undefined`, the other taps will not execute, just equivalent to `ignored`. But in rspack, each scanner runs independently.